src/book/mod.rs

```diff
@@ -519,6 +519,14 @@ impl MDBook {
         &[]
     }
 
+    pub fn uses_ace(&self) -> bool {
```

Comment:
Hi we have these properties already in `BookConfig` I would suggest using only those and removing these two methods